src/Services/PatientService.php

```diff
@@ -211,6 +211,7 @@ public function insert($data)
      */
     public function databaseUpdate($data)
     {
+        error_log("$$$ databaseupdate called $$$");
```

Comment:
Either remove this line or turn it into a debug as I mentioned in my other comment.